### DISCUSSION TITLE
Schema change detection

### DISCUSSION THREAD

**Discussion by alberttwong** [slingdata-io/sling-cli#296] [Ideas]
## Feature Description

Schema change detection.    Of course, it'll fail so the question is many continuous testing?

**Comment by bvallier** [comment]
Is this related to full-refresh mode where an existing schema is present and a new one (e.g. an additional column added) is being materialized? @flarco the default behavior seems to fail in this case, which i didn't expect. I thought the table would be wiped and any new columns would be added to new table, but that didn't seem to be happening. Is this expected?

**Comment by flarco** [comment]
hey Brice, if it is full refresh, it should recreate the table with the new column. If incremental, it should add the new column (and not fail). Can you paste your log output with debug mode here?

**Comment by bvallier** [comment]
my apologies - looking closer, this is a dagster issue using SnowflakePandasIOManager...
